src/package_tracking/launch/cameras.launch

```diff
@@ -0,0 +1,44 @@
+<launch>
+    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
+        <arg name="filters"           default="pointcloud"/>
+        <arg name="camera" value="one"/>
+        <arg name="serial_no" value="935722061157" />
+<!--        <arg name="enable_infra2" value = "false" />-->
+<!--        <arg name="depth_fps" value="30"/>-->
+<!--        <arg name="infra_fps" value="30"/>-->
+<!--        <arg name="color_fps" value="30"/>-->
+<!--        &lt;!&ndash; <arg name="fisheye_width" value="1280"/>-->
```

Comment:
Why is it uncommented?

Reply:
Actually, I was in gassy and I got the file from one friends and I had to make the video by the end of the day. I have commented most of the fast since my tiles already have these features enabled by default.. Also, they were different launch file for camera